internal/veracode-api-wrapper/updatemitigationinfo/README.md

```diff
@@ -0,0 +1,48 @@
+**Veracode API Wrapper - Update Mitigation Info**
+
+Calls the Veracode api to update the mitigation comment of a build id. This is effectively used in updating the comment 
```

Comment:
```suggestion
Calls the Veracode API to update the mitigation comment of a build ID. This is effectively used in updating the comment 
```

Reply:
In MD files, do not wrap the lines manually. GitHub will take care of the wrapping. Example: https://github.com/ballerina-platform/ballerina-lang/edit/master/README.md